pubnub-gson/build.gradle.kts

```diff
@@ -7,10 +7,10 @@ plugins {
 }
```

Comment:
Why there is:
id("pubnub.dokka")
instead of alias ?

Reply:
our plugins defined in `<root>/gradle-plugins` are used with `id` for now. We'd have to add all of them to `libs.versions.toml` to use `alias` and there's little benefit because they don't need a "version"